tools/serverpod_cli/lib/src/commands/generate.dart

```diff
@@ -56,21 +56,24 @@ class GenerateCommand extends ServerpodCommand {
 
     var endpointsAnalyzer = EndpointsAnalyzer(config);
 
-    bool hasErrors = await performGenerate(
-      config: config,
-      endpointsAnalyzer: endpointsAnalyzer,
+    bool success = await log.progress(
+      'Generating project',
```

Comment:
Text is not exactly accurate as we aren't generating a project. Perhaps a better message would be `Generating code`.

Reply:
Good catch! 👍